src/client/components/map-component.tsx

```diff
@@ -0,0 +1,97 @@
+const botIcon = new L.Icon({
+    iconUrl: botSvg,
+    iconRetinaUrl: botSvg,
+    popupAnchor:  [-0, -0],
+    iconSize: [45,52],     
+});
+
+const coffeeUrl = 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9ImN1cnJlbnRDb2xvciIgc3Ryb2tlLXdpZHRoPSIyIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIGNsYXNzPSJsdWNpZGUgbHVjaWRlLWNvZmZlZSI+PHBhdGggZD0iTTEwIDJ2MiIvPjxwYXRoIGQ9Ik0xNCAydjIiLz48cGF0aCBkPSJNMTYgOGExIDEgMCAwIDEgMSAxdjhhNCA0IDAgMCAxLTQgNEg3YTQgNCAwIDAgMS00LTRWOWExIDEgMCAwIDEgMS0xaDE0YTQgNCAwIDEgMSAwIDhoLTEiLz48cGF0aCBkPSJNNiAydjIiLz48L3N2Zz4=';
+const coffeeIcon = new L.Icon({
+    iconUrl: coffeeUrl,
+    iconRetinaUrl: coffeeUrl,
+    popupAnchor:  [-0, -0],
+    iconSize: [32,45],     
+});
+
+const plugUrl = 'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0IiBmaWxsPSJub25lIiBzdHJva2U9ImN1cnJlbnRDb2xvciIgc3Ryb2tlLXdpZHRoPSIyIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIGNsYXNzPSJsdWNpZGUgbHVjaWRlLXBsdWciPjxwYXRoIGQ9Ik0xMiAyMnYtNSIvPjxwYXRoIGQ9Ik05IDhWMiIvPjxwYXRoIGQ9Ik0xNSA4VjIiLz48cGF0aCBkPSJNMTggOHY1YTQgNCAwIDAgMS00IDRoLTRhNCA0IDAgMCAxLTQtNFY4WiIvPjwvc3ZnPg==';
+const plugIcon = new L.Icon({
+    iconUrl: plugUrl,
+    iconRetinaUrl: plugUrl,
+    popupAnchor:  [-0, -0],
+    iconSize: [32,45],     
+});
+
+const log = getLogger('MapComponent');
+log.setLevel('debug');
+
+
+
```

Comment:
Doc comment please, explaining what this component does.

Reply:
done